### PR TITLE
workup -> workups

### PR DESCRIPTION
**Pull request by skearnes**
Repeated fields should have plural names.

### CODE REVIEW

File: examples/2_Nielsen_Deoxyfluorination_Screen/example_nielsen.ipynb [hunk 1]
```diff
@@ -1558,7 +1558,7 @@
...
+       "'To a 1 mL glass vial was added (1) 0.1 mmol as a limiting reactant + 125 μL THF as a solvent (dried); (2) 0.11 mmol as a reactant + 125 μL THF as a solvent (dried); and (3) 0.15 mmol MTBD as a reagent (purchased from Sigma-Millipore). The reaction was run under ambient temperature conditions. The reaction was run under air after fully sealing the reaction vessel. The reaction mixture was stirred at a high rate (600 rpm) using a stir bar. The reaction was analyzed after 48 h by NMR (other) (19F NMR using 1 equiv 1-fluoronaphthalene in 250 uL chloroform as internal standard)., the desired product, was observed with a yield of 54.000% (± 4.800%)..'"
```
Comment (skearnes) [inline examples/2_Nielsen_Deoxyfluorination_Screen/example_nielsen.ipynb:1561]: Need to fix this.

File: examples/1_Santanilla_Nanomole_HTE/example_Santanilla.ipynb [hunk 2]
```diff
@@ -6345,27 +6345,27 @@
...
+      "Reaction entry 910:\n"
      ]
     },
     {
      "data": {
       "text/plain": [
        "identifiers {\n",
        "  type: NAME\n",
-       "  value: \"Buchwald-Hartwig thioetherification\"\n",
+       "  value: \"Buchwald-Hartwig etherification\"\n",
```
Comment (skearnes) [inline examples/1_Santanilla_Nanomole_HTE/example_Santanilla.ipynb:6356]: Something isn't right here...

Comment (michaelmaser) [reply examples/1_Santanilla_Nanomole_HTE/example_Santanilla.ipynb:6356]: What's wrong?

Comment (skearnes) [reply examples/1_Santanilla_Nanomole_HTE/example_Santanilla.ipynb:6356]: Oh these are randomly selected; nevermind.

File: examples/2_Nielsen_Deoxyfluorination_Screen/example_nielsen.ipynb (hunk 1 above)
Comment (skearnes) [reply examples/2_Nielsen_Deoxyfluorination_Screen/example_nielsen.ipynb]: Done.

**Review by connorcoley — APPROVED** [review]

**Pull request merged**